packages/block-editor/src/components/responsive-block-control/README.md

```diff
@@ -25,7 +25,7 @@ import { useState } from 'react';
 import { registerBlockType } from '@wordpress/blocks';
 import {
 	InspectorControls,
-	ResponsiveBlockControl,
+	__experimentalResponsiveBlockControl as ResponsiveBlockControl,
```

Comment:
Minor fix: While preparing this PR, I realized that this code example does not work. This component is [exported with a __experimental prefix](https://github.com/WordPress/gutenberg/blob/0b5cd48bfb474377e6947a7c3302d37bd910a586/packages/block-editor/src/components/index.js#L82).